### PR TITLE
Add FAQ links for individual extensions in list

### PR DESCRIPTION
**Pull request by mezzode**
See #841

### CODE REVIEW

**Comment by Soitora** [comment]
I think #841 was referring to the FAQ/Changelog entries here https://github.com/tachiyomiorg/tachiyomi-extensions/tree/master/src/all/batoto

**Comment by arkon** [comment]
Indeed it is. The index JSON indicates if it exists.

**Comment by mezzode** [comment]
I updated the PR to use the index JSON, but it seems like the index JSON is including some extensions that don't have folders in the tachiyomi-extensions repo as having readmes e.g. Nocturnal scans, Asura scans, miauscan?

File: src/.vuepress/components/ExtensionItem.vue [hunk 1]
```diff
@@ -18,6 +18,10 @@ export default {
 		apkUrl: function () {
 			return `https://raw.githubusercontent.com/tachiyomiorg/tachiyomi-extensions/repo/apk/${this.item.apk}`;
 		},
+		faq: function () {
+			if (!this.item.hasReadme) return undefined;
+			return `https://github.com/tachiyomiorg/tachiyomi-extensions/blob/master/src/${this.pkgId.replace('.', '/')}`;
```
Comment (arkon) [inline src/.vuepress/components/ExtensionItem.vue:23]: It's actually a bit more complicated than that -- https://github.com/tachiyomiorg/tachiyomi/blob/master/app/src/main/java/eu/kanade/tachiyomi/ui/browse/extension/details/ExtensionDetailsPresenter.kt#L124-L134

Comment (mezzode) [reply src/.vuepress/components/ExtensionItem.vue:23]: Ok I can build the url for the multisrc packages but I'm not seeing a way to distinguish which ones need it from the index.json, so I took a stab at making a PR to add it tachiyomiorg/tachiyomi-extensions#14023 but I'm not sure how to run a build using my branch for testing it

**Comment by mezzode** [comment]
Blocked on https://github.com/tachiyomiorg/tachiyomi-extensions/pull/14023

**Comment by mezzode** [comment]
P.S. I started contributing to Tachiyomi cuz of Hacktoberfest but since tachiyomiorg/tachiyomi-extensions#14023 seems to be more problematic than I expected I don't think this will be merged by the deadline of the end of Oct 31, so if it's not too much trouble I'd appreciate it if you could add the hacktoberfest-accepted label to the PR before then in the meantime so I can still get my free shirt 😅

**Comment by Fatahbedjbedj** [comment]
Please add a site 
https://mnhaestate.com/?search=trinity+wonder

**Comment by Soitora** [comment]
Since your blocked PR https://github.com/tachiyomiorg/tachiyomi-extensions/pull/14023 got inactive, then this should be assumed inactive too unless you revive it.

**Pull request closed without merging**